### PR TITLE
Update 05-steve-kalmbach-kellogg.md

### PR DESCRIPTION
**Pull request by mchittu**
Need to pull branch created by Steve for Project xyz

### CODE REVIEW

File: _slides/05-steve-kalmbach-kellogg.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+hello world
```
Comment (beardofedu) [inline _slides/05-steve-kalmbach-kellogg.md:6]: Great change! Might want to reference the image for your slide though 👍

**Review by mchittu — COMMENTED** [review]
Awesome work

File: _slides/05-steve-kalmbach-kellogg.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![kimonotocat](https://octodex.github.com/images/kimonotocat.png)
```
Comment (beardofedu) [inline _slides/05-steve-kalmbach-kellogg.md:8]: If you want a new image, get it from [here](https://octodex.github.com)

File: _slides/05-steve-kalmbach-kellogg.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _slides/05-steve-kalmbach-kellogg.md:3]:
```suggestion
title: "Welcome to steve-kalmbach-kellogg's slide!"
```

**Review by beardofedu — CHANGES_REQUESTED** [review]
This is looking good, one suggestion for you to implement and maybe think about a new caption 👍

**Review by beardofedu — APPROVED** [review]

**Pull request merged**